### PR TITLE
Refactored to remove unnecessary queries

### LINKED ISSUE #1: edit-ideas.js - Uncached jQuery selectors
- edit-ideas.js 3-5: See three references to $(this). This is unnecessarily querying three times when a reference to $(this) could be stored.

### PR DESCRIPTION
**Pull request by jwperry**
Closes #1 

@edgarduran - Let me know if you agree with the refactor below - thanks!
@rrgayhart - Tagging you in for Refactor Tractor exercise.
#### What's this PR do?
- This PR removes unnecessary queries, storing the query reference for repeated use instead.
#### Where should the reviewer start?
- Review lines 3-6 of edit-ideas.js.
#### How should this be manually tested?
- This may be manually tested by starting up the local server and simply attempting to edit an idea.
#### Any background context you want to provide?
- This is definitely an inefficiency that was repeated several times in this project. As it was the first thing I've built entirely in JavaScript, I definitely did not have a full understanding at the time of how to use jQuery efficiently.
#### Screenshots (if appropriate)
- N/A
#### What gif best describes this PR or how it makes you feel?

This

### CODE REVIEW

**Comment by edgarduran** [comment]
Looks good! Way to go, great idea to refactor this.

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,19 @@
+0 info it worked if it ends with ok
```
Comment (rrgayhart) [inline npm-debug.log]: This should be removed before you merge this PR

File: coverage/.resultset.json [hunk 2]
```diff
@@ -101,6 +101,6 @@
         null
       ]
     },
-    "timestamp": 1458804067
+    "timestamp": 1458808415
```
Comment (rrgayhart) [inline coverage/.resultset.json]: Do you need these files to be committed in source code?

**Comment by jwperry** [comment]
@rrgayhart Almost certainly not. Honestly, I'm not sure why those got pulled in. But I agree, will remove those from PR before merging.

**Pull request merged**